TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0"
+          "version": "1.0.0-servicing-004500"
```

Comment:
@ericstj, I think the only other option to workaround the NuGet issue is to not downgrade the versioning for servicing --- i.e., servicing is `1.0.101`, `1.0.102` and marking master (which is currently `1.0.1`) as `1.1.0` to not conflict with servicing.